src/webgpu/util/f32_interval.ts

```diff
@@ -1867,3 +1882,24 @@ const TruncIntervalOp: PointToIntervalOp = {
 export function truncInterval(n: number | F32Interval): F32Interval {
   return runPointToIntervalOp(toF32Interval(n), TruncIntervalOp);
 }
+
+/** Once-allocated ArrayBuffer/views to avoid overhead of allocation when converting between numeric formats */
+const unpack4x8unormData = new ArrayBuffer(4);
```

Comment:
Might change this to unpackData* in the upcoming PRs, because I don't think the various unpack*Interval calls should be calling each other, so only one ArrayBuffer and set of views is needed